			     BASH PATCH REPORT
			     =================

Bash-Release:	5.2
Patch-ID:	bash52-026

Bug-Reported-by:	Stefan Klinger <readline-gnu.org@stefan-klinger.de>
Bug-Reference-ID:
Bug-Reference-URL:	https://lists.gnu.org/archive/html/bug-readline/2023-08/msg00018.html

Bug-Description:

The custom color prefix that readline uses to color possible completions
must have a leading `.'.

Patch (apply with `patch -p0'):

*** ../bash-5.2-patched/lib/readline/colors.c	2021-12-08 11:38:25.000000000 -0500
--- lib/readline/colors.c	2023-08-28 16:40:04.000000000 -0400
***************
*** 74,78 ****
  static void restore_default_color (void);
  
! #define RL_COLOR_PREFIX_EXTENSION	"readline-colored-completion-prefix"
  
  COLOR_EXT_TYPE *_rl_color_ext_list = 0;
--- 74,78 ----
  static void restore_default_color (void);
  
! #define RL_COLOR_PREFIX_EXTENSION	".readline-colored-completion-prefix"
  
  COLOR_EXT_TYPE *_rl_color_ext_list = 0;

*** ../bash-5.2/patchlevel.h	2020-06-22 14:51:03.000000000 -0400
--- patchlevel.h	2020-10-01 11:01:28.000000000 -0400
***************
*** 26,30 ****
     looks for to find the patch level (for the sccs version string). */
  
! #define PATCHLEVEL 25
  
  #endif /* _PATCHLEVEL_H_ */
--- 26,30 ----
     looks for to find the patch level (for the sccs version string). */
  
! #define PATCHLEVEL 26
  
  #endif /* _PATCHLEVEL_H_ */
